app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/StartController.scala

```diff
@@ -73,26 +74,26 @@ class StartController @Inject()(
         } yield knownFactsResultOpt
       } else Future successful None
 
-      val decideOnSubscription = knownFactsResultOpt.flatMap {
+      knownFactsResultOpt.flatMap {
         case Some(knownFacts) =>
           for {
             _                   <- sessionStoreService.cacheKnownFactsResult(knownFacts)
             subscriptionProcess <- subscriptionService.getSubscriptionStatus(knownFacts.utr, knownFacts.postcode)
```

Comment:
You have done what I have created ticket for before, and has been closed. Now that we are caching isSubscribedToETMP in the knownFacts. this is the flag for isPartiallySubscribed

Reply:
subscriptionService.getSubscriptionStatus(knownFacts.utr, knownFacts.postcode)
This can be eliminated since we are caching the check for partially subscribed.